packages/framework-provider-aws-infrastructure/src/infrastructure/stacks/application-stack.ts

```diff
@@ -50,6 +50,11 @@ export class ApplicationStackBuilder {
       description: 'The base URL for all the auth endpoints and for sending GraphQL mutations and queries',
     })
 
+    new CfnOutput(stack, 'graphQL', {
```

Comment:
I would name this output with the same style as other URLs, `graphqlURL` for consistency (Other outputs that contain a URL are "httpURL" and "websocketURL")